Week1/homework/createTable.js

```diff
@@ -0,0 +1,22 @@
+var mysql      = require('mysql');
+var connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'hyfuser',
+  password : 'hyfpassword',
+  database : 'myWorld'
+});
+
+connection.connect(function(err) {
+    if (err) throw err;
+    console.log("Connected!");
+    var tables = [
+        "CREATE TABLE country (Country_ID int, Country_Name char(52) ,Continent enum('Asia','Europe','North America','Africa','Oceania','Antarctica','South America') ,Region char(50),SurfaceArea float,Population int,LifeExpectancy float, GNP float ,GNPOld float, LocalName char(50),GovernmentForm char(50),HeadOfState char(50),Capital char(50))",
```

Comment:
This is good but I would recommend you to use varchar instead of char when you have String with different size as it will take less space in the db.